docs/coordinates/galactocentric.rst

```diff
@@ -0,0 +1,108 @@
+.. _coordinates-galactocentric:
+
+==========================================
+Transforming to Galactocentric coordinates
+==========================================
+
+[sec:transform]
+
```

Comment:
Maybe start with a sentence containing
```
`~astropy.coordinates.Galactocentric`
```
so that it's easy to browse back and forth between this page and the docstring?